### PR TITLE
Create schema and models

### LINKED ISSUE #2: Create schemas and models

### PR DESCRIPTION
**Pull request by Guruprasanna02**
Closes #2

### CODE REVIEW

File: src/models.rs [hunk 1]
```diff
@@ -0,0 +1,82 @@
+#[derive(Queryable)]
+pub struct AttackType {
+    pub id: i32,
+    pub att_type: i32,
+    pub attack_radius: i32,
+    pub attack_damage: i32,
+}
+
```
Comment (TejusR) [inline src/models.rs:11]: Shouldn't you also add another struct for inserting a new record?

**Review by sakthi-prasath — DISMISSED** [review]

File: migrations/2021-12-23-175733_AOT/up.sql [hunk 2]
```diff
@@ -0,0 +1,115 @@
+-- Your SQL goes here
+
+CREATE TABLE public.user (
+	id serial NOT NULL,
+	name VARCHAR(255) NOT NULL,
+	email VARCHAR(255) NOT NULL,
```
Comment (pradeep-707) [inline migrations/2021-12-23-175733_AOT/up.sql]: make email unique

File: migrations/2021-12-23-175733_AOT/up.sql [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+) WITH (
+  OIDS=FALSE
+);
+
+CREATE TABLE public.game (
+	id serial NOT NULL,
+	attack_id serial NOT NULL,
+	defend_id serial NOT NULL,
+	map_layout_id serial NOT NULL,
+	attack_Score INTEGER NOT NULL,
```
Comment (pradeep-707) [inline migrations/2021-12-23-175733_AOT/up.sql]: s should be small

File: migrations/2021-12-23-175733_AOT/up.sql [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+	password VARCHAR(255) NOT NULL,
+	is_verified BOOLEAN NOT NULL,
+	CONSTRAINT user_pk PRIMARY KEY (id)
+) WITH (
+  OIDS=FALSE
+);
+
+CREATE TABLE public.levels_fixture (
+	id serial NOT NULL,
+	start_date serial NOT NULL,
```
Comment (pradeep-707) [inline migrations/2021-12-23-175733_AOT/up.sql]: use date type

File: migrations/2021-12-23-175733_AOT/up.sql [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+
+
+CREATE TABLE public.shortest_path (
+	base_id INTEGER NOT NULL,
+	source_x INTEGER NOT NULL,
+	source_y INTEGER NOT NULL,
+	dest_x INTEGER NOT NULL,
+	dest_y INTEGER NOT NULL,
+	pathlist VARCHAR(255) NOT NULL,
```
Comment (pradeep-707) [inline migrations/2021-12-23-175733_AOT/up.sql]: 255 might not be enough, make it 1000

**Review by pradeep-707 — CHANGES_REQUESTED** [review]

**Review by pradeep-707 — APPROVED** [review]

**Pull request merged**